pkgs/development/python-modules/fabulous/default.nix

```diff
@@ -18,7 +19,13 @@ buildPythonPackage rec {
     hash = "sha256-hchlxuB5QP+VxCx+QZ2739/mR5SQmYyE+9kXLKJ2ij4=";
   };
 
-  patches = [ ./relative_import.patch ];
+  patches = [
+    ./relative_import.patch
+    (fetchpatch2 {
+      url = "https://github.com/jart/fabulous/commit/5779f2dfbc88fd81b5b5865247913d4775e67959.patch?full_index=1";
+      hash = "sha256-miWFt4vDpwWhSUgnWDjWUXoibijcDa1c1dDOSkfWoUg=";
+    })
```

Comment:
A link to the PR here would be great to have so that people know the provenance of the patch.

Reply:
Added a comment above!

Reply:
I'm a bit uneasy with patches witch are not accepted upstream and only in a PR. if you force-push the PR after a review, the link will be broken. I'd rather go with something like https://github.com/NixOS/nixpkgs/pull/327200/files

Reply:
GitHub doesn’t garbage‐collect old commits; these links are stable indefinitely (more stable than ones that reference the PR number, actually).

Reply:
Technically you can request GH to garbage-collect, but yes that's still less likely than the PR branch's head changing.

Reply:
thank you I did not know